docs/db-model.puml

```diff
@@ -26,10 +32,12 @@ entity "**repository_configurations**" {
   *""name"": //character varying(255) //
   ""versions"": //character varying(255)[] //
   *""arch"": //character varying(255) //
```

Comment:
If the * is supposed to mean required, I don't think arch actually is.

Reply:
i believe its required in the db, but not required in the api.   In the DB its stored as empty string if not passed in  (this is auto generated puml)